### PR TITLE
Remove unused hclFailures var

### PR DESCRIPTION
**Pull request by jkodroff**

### CODE REVIEW

**Review by stack72 — APPROVED** [review]

File: pkg/tfgen/docs.go [hunk 1]
```diff
@@ -1213,19 +1210,6 @@ func (g *Generator) convertHCL(hcl, path, exampleTitle string) (string, string,
 			return fmt.Errorf("failed to convert HCL for %s to %v: %w", path, languageName, err)
 		}
 		if diags.All.HasErrors() {
-			if stderr.Len() != 0 {
```
Comment (stack72) [inline pkg/tfgen/docs.go:1216]: We really don't need these lines?

Comment (jkodroff) [reply pkg/tfgen/docs.go:1216]: We used to return `stderr.String()` in addition to the converted HCL and an error, and we would store the value in `hclFailures`, but nothing ever read from `hclFailures`. I believe removing `stderr.String()` as a return value means that writing anything to `stderr` is pointless.

Can you confirm my logic here?

Comment (jkodroff) [reply pkg/tfgen/docs.go:1216]: Confirmed that this change does not change the output nor schema.json for the GCP provider.

**Comment by jkodroff** [comment]
The Windows build is failing because of a network error. Given the small scope of this change, making a judgment call that it's safe to merge.

**Pull request merged**